src/hooks/data/concentrators/useConcentratorTargetAssets.ts

```diff
@@ -1,11 +1,17 @@
-import { ConcentratorTargetAsset } from "@/lib/types"
+import { Address } from "wagmi"
+
+import { ETH_TOKEN_ADDRESS } from "@/lib/isEthTokenAddress"
 
 // ! Mocked hook for use with new registry interface
 
 export function useConcentratorTargetAssets(options: {
-  onSuccess?: (data: ConcentratorTargetAsset[]) => void
+  onSuccess?: (data: Address[]) => void
 }) {
-  const data: ConcentratorTargetAsset[] = ["auraBAL", "ETH", "cvxCRV"]
+  const data: Address[] = [
```

Comment:
Hardcoding the target asset symbol isn't going to work because we need to retrieve details about the asset (e.g. decimals, balance, etc). 

Added a comment on the registry updates ticket to express this (https://github.com/FortressFinance/issues/issues/112#issuecomment-1436140875).